lib/dependency-manager-adapters/npm.js

```diff
@@ -68,7 +68,7 @@ module.exports = class {
         if (fs.statSync(path.join(this.cwd, this.yarnLock)).isFile()) {
           ui.writeLine(
             chalk.yellow(
-              'Detected a yarn.lock file. Add `useYarn: true` to your `config/ember-try.js` configuration file if you want to use Yarn to install npm dependencies.',
+              "Detected a yarn.lock file. Add `packageManager: 'yarn'` to your `config/ember-try.js` configuration file if you want to use Yarn to install npm dependencies.",
```

Comment:
Yes, sounds good to me, especially using the `packageManager` field in the package file. Do you want me to do this instead of this PR, or as a follow-up one, or...?